app/javascript/components/widget-transition-controls.test.jsx

```diff
@@ -14,6 +15,12 @@ describe('isPaused', () => {
   });
 });
 
+describe('msRemaining', () => {
+  it('returns true if the milliseconds remaining is any non-null value', () => {
+    expect(isStarted({ msRemainingSinceLastStart: null })).toEqual(false);
+  });
```

Comment:
So we're describing the `isStarted` method. Our `it(...)` doesn't match up with the `toEqual` assertion.

Its fine to squash the unit test into the `isStarted` addition.

Reply:
I'd like to see the unit tests in the commit that introduce `isStarted`